test/sagemaker_tests/pytorch/training/resources/neuron/mlp/requirements.txt

```diff
@@ -1 +0,0 @@
-torchvision==0.12.0
```

Comment:
can we delete the file if the contents are empty?

Reply:
Done